src/excalidraw-app/components/ExportToExcalidrawPlus.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { createIcon } from "../../components/icons";
+import { Card } from "../../components/Card";
+import { ToolButton } from "../../components/ToolButton";
+import { serializeAsJSON } from "../../data/json";
+import { getImportedKey, createIV, generateEncryptionKey } from "../data";
+import { loadFirebaseStorage } from "../data/firebase";
+import { NonDeletedExcalidrawElement } from "../../element/types";
+import { AppState } from "../../types";
+import { nanoid } from "nanoid";
+import { t } from "../../i18n";
+
+const excalidrawPlusIcon = createIcon(
```

Comment:
let's move this to the icons file?

Reply:
This file should be moved to excalidraw-app folder as its part of app

Reply:
> This file should be moved to excalidraw-app folder as its part of app

It is already there :)

> let's move this to the icons file?

Didn't want to add it to icons because it's specific to excalidraw.com

Reply:
let's create a icons file inside `excalidraw-app` then?